docs/howto/edit-in-protege.md

```diff
@@ -0,0 +1,74 @@
+# Editing a term in protege
+
+Before you start:
+
+-   make sure you are working on a [here](https://docs.google.com/presentation/d/1M8NZQOIQVswng-so6ROxVeMJfDnzth7BYNj_5MXxEik/edit#slide=id.g9db6baf776_1_0).
+-   make sure you have the editor's file open in Protege (in ODK ontologies, located in: ./src/ontology/ONT-edit.owl) where ONT is the name of your ontology (eg mondo-edit.owl for MONDO)
+
+## Adding, editing, annotating and deleting axioms
+
```

Comment:
Maybe add some more general text here on 

- Protege views & Tabs. 
- Editing icons: + o, @ x
- class expression editor & autocomplete (this might belong on text on editing logical axioms)